scripts/userstyles.yml

```diff
@@ -281,6 +284,22 @@ userstyles:
     readme:
       app-link: "https://crowdin.com"
     current-maintainers: [*ryanccn]
+  crunchyroll:
+    name: Crunchyroll
+    categories: [entertainment]
+    icon: crunchyroll
+    color: peach
+    readme:
+      app-link: "https://crunchyroll.com"
+      faq:
+        - question: "What are the player options for?"
+          answer: |
+            The player options allow you to specify how the catppuccin colors are applied to the video player:
+
+            **Player: Flavor** - Which flavor of catppuccin to use for the player. *Auto* uses the same flavor as the rest of the site.
+            **Player: Setting** - To what extend the catppuccin theme is used. *None*: disables theming of the video player, *Minimal*: background, timeline (accent only) and loading circle, *Medium*: same as minimal plus menus, *Full*: same as medium plus all controls (contrast is suboptimal).
+            **Player: Background** - When to apply theming to the background/black edges. *Not Fullscreen* applies the theme to the background when the player is not in fullscreen mode.
```

Comment:
```suggestion
            The player options allow you to specify how the Catppuccin colors are applied to the video player:

            **Player: Flavor** - Which flavor to use for the player. *Auto* uses the same flavor as the rest of the site.
            **Player: Setting** - To what extent the player is themed. *None* disables theming of the video player; *Minimal* themes the background, timeline, and loading circle; *Medium* is the same as *Minimal* with the addition of themed menus, *Full*: same as *Medium* with the addition of all controls (contrast is suboptimal).
            **Player: Background** - When to apply theming to the background/black edges. *Not Fullscreen* applies the theme to the background when the player is not in fullscreen mode.
```

Reply:
`timeline (accent only)` - Only the original orange is replaced by the accent color atm. The background and color for the cached part are only replaced in "full".

Reply:
Sure, that wasn't clear by your original text. I'll update it.